aml-development/frontend
# Contact owners from listing

Adds functionality for user to send message to the owner of the listing.

Closes #AMLOS-491

Description
Allows user to send a message to Listing owners from the Listing page.

Return to Owner Feedback
Please provide feedback for the listing owner about what they should do to make this listing ready for publication

Acceptance Criteria:

Users can contact the owners of a listing from the Listing Page, but clicking a message icon next to Owners on the left side of a listing.

How to Test
Pull this branch
log in as bigbrother and julia and wsmith.
use the button on the left to send a notification.
Check julia and wsmith receive the notification.
log into other stewards and owners to check for the notification.

Closes AMLOS-665
